intel_extension_for_transformers/neural_chat/examples/finetuning/dpo_pipeline/dpo_clm.py

```diff
@@ -216,6 +216,10 @@ def find_all_linear_names(model):
 
     model_args, data_args, training_args, finetune_args = parser.parse_args_into_dataclasses()
 
+    if training_args.use_cpu:
```

Comment:
qlora on Xeon does not show perf benefits

Reply:
quant/dequant takes more time

Reply:
Then we should remove it

Reply:
done